app/views/alaveteli_pro/info_requests/_embargo_info.html.erb

```diff
@@ -1,23 +1,25 @@
-<p class="form_item_note--important js-embargo-active">
-  <% if tense == :future %>
-    <%= _('This request will be private on {{site_name}} until ' \
-          '<span class="js-embargo-expiry">{{embargo_publish_at}}</span>',
-          site_name: AlaveteliConfiguration.site_name,
-          embargo_publish_at: embargo_publish_at(embargo)) %>
-  <% else %>
-    <%= _('This request is private on {{site_name}} until ' \
-          '<span class="js-embargo-expiry">{{embargo_publish_at}}</span>',
-          site_name: AlaveteliConfiguration.site_name,
-          embargo_publish_at: embargo_publish_at(embargo)) %>
-  <% end %>
-</p>
-
-<span class="js-embargo-inactive"<% if embargo && embargo.publish_at %> style="display: none;"<% end %>>
-  <% if tense == :future %>
-    <p><%= _("Unless you choose a privacy option, your request will be " \
-             "public on {{site_name}} immediately.",
-             site_name: AlaveteliConfiguration.site_name) %></p>
-  <% else %>
-    <p><%= _("This request is public") %></p>
-  <% end %>
-</span>
+<% if embargo && embargo.publish_at %>
+  <p class="js-embargo-active">
+    <% if tense == :future %>
+      <%= _('This request will be private until ' \
+            '<span class="js-embargo-expiry"><strong>{{embargo_publish_at}}</strong></span>',
+            site_name: AlaveteliConfiguration.site_name,
+            embargo_publish_at: embargo_publish_at(embargo)) %>
+    <% else %>
+      <%= _('This request is private until ' \
+            '<span class="js-embargo-expiry"><strong>{{embargo_publish_at}}</strong></span>',
```

Comment:
I've also just noticed that this sneaks in `<strong>` tags. Again, lets do that in a separate commit.

Reply:
Fixed in cacf82102